zeus_arm:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  joint_1_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint_1
  #  pid: {p: 27.0, i: 6.0, d: 17.0}
  joint_2_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint_2
  #  pid: {p: 700.0, i: 300.0, d: 100.0}
  joint_3_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint_3
  #  pid: {p: 750.0, i: 90.0, d: 150.0}
  joint_4_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint_4
  #  pid: {p: 150.0, i: 15.0, d: 20.0}
  joint_5_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint_5
  #  pid: {p: 10.0, i: 5.0, d: 0.0}

  gazebo_ros_control/pid_gains:
    joint_1 : {p: 150.0, i: 15.0, d: 0.0}
    joint_2 : {p: 30.0, i: 5.0, d: 0.0}
    joint_3 : {p: 30, i: 0.0, d: 0.0}
    joint_4 : {p: 1.0, i: 0.5, d: 0.0}
    joint_5 : {p: 0.01, i: 0.0, d: 0.0}

